apps/ledger-live-mobile/src/families/tron/VoteFlow/02-VoteCast.tsx

```diff
@@ -221,7 +221,7 @@ export default function VoteCast({ route, navigation }: Props) {
                   }}
                 >
                   <LText semiBold style={styles.availableAmount}>
-                    text
+                    PLACEHOLDER_TEXT
```

Comment:
I guess I should know what's this coin UI is doing but... do you know what this is for, and how it is used?

Reply:
This is part of a Trans component and it's used for string interpolation, allowing you to use bold text in the middle of the sentence. The placeholder text is irrelevant, but the rule I introduced for untranslated strings requires them to be uppercase, that's why I replaced these.